Sources/App/Commands/Analyzer.swift

```diff
@@ -220,7 +220,7 @@ func updateVersion(on database: Database, version: Version, manifest: Result<Man
     switch manifest {
         case .success(let manifest):
             version.packageName = manifest.name
-            version.swiftVersions = manifest.swiftLanguageVersions?.compactMap(SemVer.parse) ?? []
+            version.swiftVersions = manifest.swiftLanguageVersions ?? []
```

Comment:
Due to our stricter SemVer parsing I've had to change saving swift language versions as plain strings. They're not really SemVer - 5.0, 5.1 ... I don't think it makes a difference really as we're not using the actual version info anyway other than display (where 5.0 will read better than 5.0.0)

Reply:
For the Swift version, I agree! Apple don't refer to Swift 5.2.0, they refer to 5.2. I made the same decision in the SwiftPM Library implementation.

Just a note while we're on the subject. I think we should show a full, strict semantic version for *packages* everywhere, even if it's a .0 patch. Xcode *only* sees full semantic versions (including a patch) as valid, and we should do the same.

Reply:
Great, we're on the same page then!

Re SemVer for packages - that's actually a great point. In the PR after this one you'll see that I've collapsed branches and tags into `References`. Currently those are `.branch(String)` and `.tag(String)` but given that I'll change that to `.tag(SemVer)` then.

Does that make sense?

Reply:
Yep, I saw the column change and it looks good, and it's even better if it's a `tag(SemVer)` 👍